common/test/metadata/MetaDataMatcher.scala

```diff
@@ -40,4 +40,26 @@ object MetaDataMatcher extends Matchers  {
 
   }
 
+  def ensureDeepLink(result: Future[Result], expectedLink: String) {
+    val body = Jsoup.parseBodyFragment(contentAsString(result))
+    status(result) should be(200)
+    val script = body.getElementsByAttributeValue("href", expectedLink)
+    script.size() should be(1)
+  }
+
+  def ensureNoDeepLink(result: Future[Result], nonExistantLink: String) {
+    val body = Jsoup.parseBodyFragment(contentAsString(result))
+    status(result) should be(200)
+    val script = body.getElementsByAttributeValue("href", nonExistantLink)
+    script.size() should be(0)
```

Comment:
@johnduffell yeah, that's what I am doing. The argument nonExistantLink is the the thing which starts with app://., This method just saves me from having to look for things starting with 'app;//' in four different places. see changes to IndexMetaDataTest above .